livre/Eq/Config.cpp

```diff
@@ -253,9 +279,13 @@ bool Config::init()
     subscriber->registerHandler( zeq::hbp::EVENT_LOOKUPTABLE1D,
                                  boost::bind( &detail::Config::onLookupTable1D,
                                               _impl, _1 ));
-
-    SubscriberPtr rcSubscriber( new zeq::Subscriber(
-                                    lunchbox::URI( "restcmd://" )));
+#ifdef LIVRE_USE_RESTCONNECTOR
+    std::string subscriberSchema = _impl->framedata.getNetworkParameters()->zeqSchema
+                                   + SUBSCRIBER_SCHEMA_SUFFIX;
+    SubscriberPtr rcSubscriber( new zeq::Subscriber( lunchbox::URI( subscriberSchema ) ) );
+#else
+    SubscriberPtr rcSubscriber( new zeq::Subscriber( lunchbox::URI( "restcmd://" ) ) );
```

Comment:
It is not so trivial... For example when it receive zeq::camera event the hbp:// subscriber  use onHBPCamera() which is tuned for RTneuron with the scaling based on volumeBBox. The vwsSubscriber use the matrix as it is. I think we first need to define a better camera event if we want to use the same subscriber.
I created a story: BVWS-230